### PR TITLE
Adds input data for hiwiki (10k revisions)

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2910,7 +2930,7 @@ datasets/ptwiki.autolabeled_revisions.10k_2020.json: \
 		datasets/ptwiki.sampled_revisions.10k_2020.json
 	cat $< | \
 	./utility autolabel --host=https://pt.wikipedia.org \
-		--trusted-groups=bot,sysop,bureaucrat \
+		--trusted-groups=bot,sysop,bureaucrat,autoreviewer,rollbacker \
```
Comment (halfak) [inline Makefile:2933]: Not sure why this got picked up by the makefile generator, but it looks right to me.

Comment (he7d3r) [reply Makefile:2933]: Good to know there is such a generator! I though the make files were created manually...

File: Makefile [hunk 2]
```diff
@@ -1907,6 +1909,24 @@ hewiki_tuning_reports: \
...
+############################# Hindi Wikipedia ################################
+# From https://quarry.wmflabs.org/query/44835
+datasets/hiwiki.sampled_revisions.10k_2020.json:
+	wget -qO- https://quarry.wmflabs.org/run/467016/output/0/json-lines > $@
+
+datasets/hiwiki.autolabeled_revisions.10k_2020.json: \
+		datasets/hiwiki.sampled_revisions.10k_2020.json
+	cat $< | \
+	./utility autolabel --host=https://hi.wikipedia.org \
+		--trusted-groups=sysop,oversight,bot,rollbacker,checkuser,abusefilter,bureaucrat,autopatrolled,autoreviewer,reviewer \
```
Comment (he7d3r) [inline Makefile]: Shouldn't `autoreviewer` be replaced by `autoreview`? 
https://hi.wikipedia.org/wiki/Special:ListUsers?group=autoreview&uselang=en

Comment (halfak) [reply Makefile]: Good catch.  Just pushed a fix and luckily it didn't change the data I pushed to Wikilabels.  I'm guessing autoreview has enough overlap with another group that it didn't matter.

**Comment by halfak** [comment]
@haksoat

**Pull request merged**